_posts/en/newsletters/2022-10-19-newsletter.md

```diff
@@ -0,0 +1,308 @@
+    both [LND][lnd 0.15.2-beta] (as mentioned in [last week's
+    newsletter][news221 lnd]) and [BTCD][btcd 0.23.2].  All users of
+    BTCD and LND should upgrade.
+
+    Until a user upgrades their software, they will suffer the
+    lack-of-confirmation problems described above and may also be
+    vulnerable to several attacks.  Some of those attacks require access
+    to significant hash rate (making them expensive and, hopefully,
+    impractical in this case).  Other attacks, particularly those
+    against LND users, require the attacker to risk losing some of their
+    funds in a channel, which is also hopefully a sufficient deterrent.
+    We again recommend upgrade and, further, we recommend that anyone
+    using any Bitcoin software sign up for security announcements from
+    that software's development team.
+
+    After the above disclosures, Loki Verloren [posted][verloren limits]
+    to the Bitcoin-Dev mailing list to suggest that direct limits be
+    added to taproot's witness size.  Greg Sanders [replied][sanders
+    limits] to note that adding limits now would not only increase code
+    complexity but could also lead to people losing their money if they
+    already received bitcoins to a script which requires a large witness
+    to spend.
+
+- **Transaction replacement option:** as reported in Newsletters
+  [#205][news205 rbf] and [#208][news208 rbf], Bitcoin Core merged
+  support for a `mempoolfullrbf` configuration option which defaults to
+  the existing Bitcoin Core behavior of only allowing [RBF
+  replacement][topic rbf] of transactions containing the [BIP125][]
+  signal.  However, if a user sets the new option to true, their node
+  will accept and relay replacement transactions that don't contain the
+  BIP125 signal, provided the replacement transactions follow all of
+  Bitcoin Core's other rules for replacements.
+
+    Dario Sneidermanis [posted][sne rbf] to the Bitcoin-Dev mailing that
+    this new option may be concerning for services which currently accept
+    unconfirmed transactions as final.  Although it's been possible for
+    years for users to run non-Bitcoin Core software (or patched
+    versions of Bitcoin Core) that allow unsignaled *full*[^full-rbf]
+    transaction replacement, there's little evidence for that software
+    being widely used.  Sneidermanis believes an easily accessible
+    option in Bitcoin Core might change that by allowing enough users
+    and miners to enable full RBF and make unsignaled replacement
+    reliable.  Reliable unsignaled replacement would also make it
+    reliable to steal from services that accept unconfirmed transactions
+    as final, requiring those services to change their behavior.
+
+    Sneidermanis also proposed an alternative: remove the configuration
+    option from the upcoming Bitcoin Core release but also add code that
+    will enable full RBF by default at a future moment.  Anthony Towns
```

Comment:
I used "future moment" because Sneidermanis proposed a height-based mechanism and Towns proposed a time-based mechanism.  I don't think we need to go into that detail in the newsletter, but I also don't want to inaccurately describe Sneidermanis's proposal by saying "date", so I chose a term I thought would generalize to both approaches.